cookbooks/aws-parallelcluster-environment/templates/cfn_bootstrap/cfn-hup-runner.sh.erb

```diff
@@ -9,6 +9,10 @@ set -ex
 [ -f /etc/profile.d/proxy.sh ] && . /etc/profile.d/proxy.sh
 
 while true; do
-  <%= @cfn_bootstrap_virtualenv_path %>/bin/cfn-hup --no-daemon --verbose
+  if [[ $(ps aux | grep "cfn-hup --no-daemon --verbose" | wc -l) -gt 1 ]]; then
```

Comment:
I suspect this is wrong.
i.e. the command stucks at line 15, and will never loop back to reach the check

Reply:
Yes you are right. Will change the implementation